app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -38,6 +38,9 @@ def refresh
             post_refresh_ems_cleanup(ems, targets)
           end
 
+          # We need to reload the ems, since it can try to autosave a models that did not pass a validation, and the
+          # ems status with the error would not save.
+          ems.reload
```

Comment:
what I dont like here is the `.reload` workaround.

How about using `.update_columns` ? This intentionally skips validations and I think we are only interested in the `last_refresh_error` and `_date` columns here and not the `updated_at` column, right?